eras/conway/impl/cddl-files/conway.cddl

```diff
@@ -147,13 +147,13 @@ vote = 0 .. 2
 
 gov_action_id =
   [ transaction_id   : $hash32
-  , gov_action_index : uint
+  , gov_action_index : uint .size 4
```

Comment:
Could you also switch GovActionIx to `Word16`, since there is no chance of having more than 65536 governance actions in a single transaction
```suggestion
  , gov_action_index : uint .size 2
```